Gemfile.jruby-1.9.lock.release

```diff
@@ -166,9 +166,9 @@ GEM
       jls-grok (~> 0.11.1)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       logstash-patterns-core
-    logstash-codec-netflow (3.10.0)
+    logstash-codec-netflow (3.11.0)
       bindata (>= 1.5.0)
-      logstash-core-plugin-api (>= 1.60, <= 2.99)
+      logstash-core-plugin-api (~> 2.0)
```

Comment:
Why do we no longer support 1.6+ plugins here?

Reply:
I guess because netflow codec did that change.
https://github.com/logstash-plugins/logstash-codec-netflow/commit/3c35fad0a00e7b1c56e4250482dfe9be3e5ca08b

Reply:
@guyboertje I think this is fine. My concern was based on a misunderstanding. Plugins that require the 1.6 version of the plugin API were broken before, and will remain that way.